app/views/publishers/vacancies/build/job_details.html.slim

```diff
@@ -21,7 +21,7 @@
           = f.govuk_text_field :job_title,
             label: { size: "s" },
             class: "govuk-input string required govuk-js-character-count",
-            hint: { text: t("helpers.hint.publishers_job_listing_job_details_form.job_title.#{vacancy.one_phase? ? vacancy.readable_phases.first : 'multiple_phases'}") },
+            hint: { text: t("helpers.hint.publishers_job_listing_job_details_form.job_title") }
```

Comment:
Now that this hint text doesn't have to do any conditions, it can go and live in the forms.yml with the rest of the hint texts! (under 'helpers', 'hint', 'publishers_job_listing_job_details_form')

Reply:
Ah, I see it is already there. You should be able to just omit the `hint` line in the view and GovUK formbuilder will find the hint text in forms.yml.

Reply:
Nice one!